test/libsolidity/util/BytesUtils.cpp

```diff
@@ -206,6 +225,19 @@ string BytesUtils::formatString(bytes const& _bytes, size_t _cutOff)
 	return os.str();
 }
 
+std::string BytesUtils::formatFixedPoint(bytes const& _bytes, size_t _fixedPointM, size_t _fixedPointN)
+{
+	solAssert(_fixedPointM != 0, "");
+	u256 max{boost::multiprecision::pow(u256{2}, static_cast<unsigned>(_fixedPointM))};
+	solAssert(u256{_bytes} >= max, "");
+	stringstream os;
+	os << u256{_bytes};
+	string value{os.str()};
```

Comment:
Padding zeroes not yet implemented. E.g. actual defined value has less digits than fractional digits.